python/config/bradley.py

```diff
@@ -0,0 +1,45 @@
+def match_containers(alma_items: list, aspace_containers: list, logger) -> tuple:
+    unmatched_alma_items = []
+    matched_aspace_containers = []
+    for item in alma_items:
+        item_id = item.get("item_data").get("pid")
+        barcode = item.get("item_data").get("barcode")
+        description = item.get("item_data").get("description")
+        alma_container_type = description.split(".")[0]
+        alma_indicator = description.split(".")[1]
+        # if indicator starts with 0, remove it
+        while alma_indicator[0] == "0":
+            alma_indicator = alma_indicator[1:]
+        # if indicator ends with " RESTRICTED", remove it
+        if alma_indicator.endswith(" RESTRICTED"):
+            alma_indicator = alma_indicator[:-11]
+        # match with ASpace top container based on container type and indicator
+        for tc in aspace_containers:
```

Comment:
As we talked about, this nested loop could be quite slow.  Instead, consider a method which builds a dictionary of ASPace data, keyed on a tuple of data needed by this config for matching Alma data, something like
```
def _get_aspace_match_data(list) -> dict[tuple]:
    # for each item
    # create "matchable" (tuple) with the ordered elements which are needed to match alma
```